braindecode/models/eegnet.py

```diff
@@ -79,8 +80,8 @@ def __init__(
         self.add_module("ensuredims", Ensure4d())
         # b c 0 1
         # now to b 1 0 c
```

Comment:
this comment is and was wrong then? maybe just remove it.

Reply:
Removed @robintibor